code/game/machinery/kitchen/microwave.dm

```diff
@@ -435,16 +435,20 @@
 	return ffuu
 
 /obj/machinery/microwave/CtrlClick(mob/user)
-    if(!user.incapacitated() && Adjacent(user) && user.dexterity_check() && anchored)
-        cook() //Cook checks for power, brokenness, and contents internally
-        return
-    return ..()
+	if(!user.incapacitated() && Adjacent(user) && user.dexterity_check() && anchored)
+		if(issilicon(user) && !attack_ai(user))
```

Comment:
Wouldn't checking `attack_ai()` force the silicon to interface with it, meaning in case of service borgs, they'll always bring up the microwave's main menu whenever they try to use one of these shortcuts?